src/main/java/org/icgc_argo/workflow_graph_lib/workflow/client/RdpcClient.java

```diff
@@ -507,6 +507,11 @@ public void onFailure(@NotNull ApolloException e) {
             clientResponse -> {
               throw new RequeueableException(EX_5XX_ERROR);
             })
-        .bodyToMono(new ParameterizedTypeReference<>() {});
+        .bodyToMono(new ParameterizedTypeReference<Map<String, Object>>() {})
+        // onErrorMap catches thrown exceptions that are not GraphExceptions (e.g. NetworkTimeout,
+        // SSLException) and wraps them with DeadLetterQueueableException
+        .onErrorMap(
+            throwable -> !(throwable instanceof GraphException),
+            throwable -> new DeadLetterQueueableException(throwable.toString()));
```

Comment:
Webclient exceptions outside of http error weren't being caught and wrapped with a GraphException which is why GraphNode couldn't resolve it. This onErrorMap should wrap any unexpected exceptions with DeadLetterQueueableException which Node can resolve.